Parsers/ASimAuditEvent/Parsers/ASimAuditEventIllumioSaaSCore.yaml

```diff
@@ -0,0 +1,392 @@
+        'workload_interface.delete', 'Workload interface deleted', 'Cloud Resource', 'Workload_interface', 'Delete',
+        'workload_interface.update', 'Workload interface updated', 'Cloud Resource', 'Workload_interface', 'Set',
+        'workload_interfaces.update', 'Workload interfaces updated', 'Cloud Resource', 'Workload_interfaces', 'Set',
+        '', 'For example, IP address changes, new interface added, and interface shut down.', 'Other', '', 'Other',
+        'workload_service_report.update', 'Workload service report updated', 'Cloud Resource', 'Workload_service_report', 'Set',
+        'workload_settings.update', 'Workload settings updated', 'Cloud Resource', 'Workload_settings', 'Set',
+        'workloads.apply_policy', 'Workloads policies applied', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.bulk_create', 'Workloads created in bulk', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.bulk_delete', 'Workloads deleted in bulk', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.bulk_update', 'Workloads updated in bulk', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.remove_labels', 'Workloads labels removed', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.set_flow_reporting_frequency', 'Workload flow reporting frequency changed', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.set_labels', 'Workload labels applied', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.unpair', 'Workloads unpaired', 'Cloud Resource', 'Workloads', 'Other',
+        'workloads.update', 'Workloads updated', 'Cloud Resource', 'Workloads', 'Set'
+    ];
+    let EventSeverityLookup = datatable(
+        severity: string,
+        EventSeverity: string
+    )
+        [
+        "err", "High",
+        "info", "Informational",
+        "warning", "Medium"
+    ];
+    let EventResultLookup = datatable(
+        status: string,
+        EventResult: string
+    )
+        [
+        "success", "Success",
+        "failure", "Failure",
+        "", "NA"
+    ];
+    let parser = (disabled: bool = false) {
+        Illumio_Auditable_Events_CL
+        | where not(disabled)
+        | lookup EventTypeLookup on event_type // fetch Object, ObjectType,EventType, Operation from lookup
+        | lookup EventSeverityLookup on severity // fetch EventSeverity from lookup
+        | lookup EventResultLookup on status // fetch EventResult from lookup
+        | extend
+            ActorUsername = case(
+                isnotnull(created_by.system), "System",
+                isnotnull(created_by.user), created_by.user.username,
+                isnotnull(created_by.agent), created_by.agent.hostname,
+                "Unknown"
+            )
+        | extend
+             ActorUsernameType = case(
+              isnotnull(created_by.system), "System",
+              isnotnull(created_by.user), "User",
+              isnotnull(created_by.agent), "Agent",
+              "Unknown"
```

Comment:
![image](https://github.com/user-attachments/assets/ee9b0d6d-4add-44ca-8d1c-f54181f2f1f0)

Reply:
please use allowed values only:
https://learn.microsoft.com/en-us/azure/sentinel/normalization-about-schemas#the-user-name